src/types.rs

```diff
@@ -134,60 +140,154 @@ impl Debug for Limits {
         if let Some(max) = self.max {
             write!(f, "={max}")?;
         }
+        if self.is_64 {
+            write!(f, "(64)")?;
```

Comment:
Maybe

```suggestion
            write!(f, "_u64")?;
```

to make it more Rust-y, like the range syntax before it.